library/alloc/src/collections/linked_list.rs

```diff
@@ -656,6 +658,8 @@ impl<T> LinkedList<T> {
     /// Provides a reference to the front element, or `None` if the list is
     /// empty.
     ///
+    /// This operation should compute in *O*(*1*) time.
```

Comment:
the "1" doesn't need to be italic.

Reply:
Unitalicized. Ready for another look when you have the time, @kennytm.